phpunit.xml.dist

```diff
@@ -14,6 +14,7 @@
         <env name="APP_DEBUG" value="1" />
         <env name="APP_SECRET" value="s$cretf0rt3st" />
         <env name="SHELL_VERBOSITY" value="-1" />
+        <env name="DATABASE_URL" value="mysql://kugis:AfAA7B63218@127.0.0.1:3306/symfonyproject"/>
```

Comment:
Cia sito gal failo netneturetu but pagalviska gite, bet tiksliai nzn, siaip px bet pasigooglink po to ar jo reik

Reply:
<env name="DATABASE_URL" value="mysql://kugis:AfAA7B63218@127.0.0.1:3306/symfonyproject"/>
šitoj eilutėje svarbu kur kugis db username tada password tada @local host ir tada dbpavadinimas, 
pridėjau į .gitignore